src/Dynamic.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import './Dynamic.css'
+
+class Dynamic extends React.Component {
+
+    constructor() {
+        super();
+        this.state = {
+            barSize: "0%",
+            dynamoVersion: "",
+            loadDescription: ""
+        }
+
+        window.setBarProperties = this.setBarProperties.bind(this);
```

Comment:
I suppose this will be accessed from Dynamo?

Reply:
Yes, that's right.

Reply:
This is great. Some more comments in this file for each function would be appreciated